### DISCUSSION TITLE
Are Multiple Sub-Products in One Model possible?

### DISCUSSION THREAD

**Discussion by Rupinder-Singh-Bell** [google/meridian#360] [Q&A] [answered]
In the Geo column we can have multiple geographies (Geo0, Geo1,... etc.) and have KPIs broken down at these levels but there is only one KPI column no "product" column so from my understanding no way to allocate that KPI to different sub-product levels. 

Ex. if i have a widget and i have 3 versions of it; basic-widget, regular-widget , and pro-widget and I would like to use units sold as the KPI how could i model all 3 widgets in one model? A customer would only purchase one of the 3 versions and my marketing ads are only for widgets in general and not the different sub-products but the 3 sub-products represent different types of customers so having one model would help to understand the effectiveness of marketing campaigns against these sub-products.

**Comment by mikewurm** [comment, marked as answer]
Meridian does not currently support multiple random effects for media coefficients. As an alternative, you might try using the geo column to represent the geo:product combination. This would give you a model with independent random effects, as opposed to the covariance structure of crossed or nested random effects - but it still offers pooling of information across geos and products.

If you try this, the `population` field should still be used to represent the geo population. Each population value will need to be repeated (once for each product).
